drive/modules/listener/watch_schedular.bal

```diff
@@ -59,16 +59,17 @@ class Job {
     public isolated function execute() {
         error? err = self.registerWatchChannel();
         if (err is error) {
-            if (self.retryCount == 1){
+            if (self.retryCount == 1 && err.message().includes(ERR_UNAUTHORIZED_WEBHOOK_CHANNEL)){
```

Comment:
If we get `ERR_UNAUTHORIZED_WEBHOOK_CHANNEL` only upon domain verification is not done, this check is fine.

Reply:
Yeah. This message [1] comes when the configured URL is not registered as a verified domain on the Google console.
[1] `const string ERR_UNAUTHORIZED_WEBHOOK_CHANNEL = "Unauthorized WebHook callback channel";`